test/appsignal/release_upgrade_test.exs

```diff
@@ -29,7 +27,6 @@ defmodule Appsignal.ReleaseUpgradeTest do
 
         until(fn ->
           assert config()[:name] == "AppSignal test suite app v2"
-          assert Nif.env_get("_APPSIGNAL_APP_NAME") == 'AppSignal test suite app v2'
```

Comment:
This is I think testing if the Nif picks up the change. If it doesn't, how do we know if the extension is using the right config for the app?

Reply:
We check the config to make sure `Appsignal.initialize/0` is called, which initializes the configuration by calling `Config.initialize/0`. The config initialization function updates the environment variables in the Nif, which is well-tested functionality. I don’t think we have to check that here again.